### ISSUE TITLE
Mystical Power Plant crashes upon difficulty selection (1.4 release)

### ISSUE THREAD

**Issue by starks** [WishMakers0/Danmakufu-Woo-Edition#6]
```
#Include file is not found[D:/mystical/script/system/Load_AllTexture.dnh].
(#includeで置換するファイル[D:/mystical/script/system/Load_AllTexture.dnh]が見つかりません)
D:/mystical/script/Th_mpp/scene/Package_PlayerSelectScene.dnh 
Package_PlayerSelectScene.dnh line(行)=1

↓
#include"../../system/Load_AllTexture.dnh"

	let csd				= GetCurrentScriptDirectory();
	
@Initialize{
	SetAutoDeleteObject(true);

	TBackground();
	TPlayerSelectScene();

	SetScriptResult("");
}

@MainLoop{
	yield;
}

@Finalize{
	PlaySEs
～～～


```

**Comment by WishMakers0** [comment]
Can you confirm whether this happens in 1.3/nf3 as well? 
Version 4 of woo doesn't work properly in a few places so I want to be sure that this isn't an issue with archives or something.

**Comment by starks** [comment]
This has occurred in 1.3 and older versions as well.

**Comment by WishMakers0** [comment]
So believe it or not, I actually figured out the issue, and it's not actually Woo related!
Sorry for _literally taking a year to address this_ (I forgot) but I investigated this and determined the cause.
MPP uses a form of relative pathing that does not work in modern ph3 and frankly, it shouldn't work in older ph3 either, but it works in the version that the game uses.  If you replace every instance of `./` with`../` at the start of an include (which I did using Find in Files with Sublime Text) then the game actually runs with minimal compatibility issues with Woo.  I plan on closing the issue once I've attached a version of the scripts that work.
